docs/major-concepts/exhaust-script.md

```diff
@@ -0,0 +1,70 @@
+---
+sidebar_position: 10
+---
+
+# Exhaust scripts
+
+Exhaust script is the major concept of the `if`, which provides two types of exhaust scripts: `if-run` and `if-csv`.
```

Comment:
if-run isn't really an exhaust script -  it does observe/regroup/compute as well. I'd define an exhaust script as a script that takes a yaml file as input and outputs some other representation of the same data.

Reply:
> if-run isn't really an exhaust script - it does observe/regroup/compute as well. I'd define an exhaust script as a script that takes a yaml file as input and outputs some other representation of the same data.

Thanks  @jmcook1186 for clarification!

Reply:
added some changes